content/documentation/getting-started/index.adoc

```diff
@@ -299,6 +299,33 @@ The login option can be specified in the Kiali CR when installing Kiali. For ins
 auth:
   strategy: openshift
 ----
+*ldap*: This option allows you to set the authentication strategy so that the user is validated using the LDAP server, then an authentication token is generated. A sample configuration for LDAP is given below
+----
+auth:
+  strategy: ldap
+  ldap:
+    # AD IP host and port
+    ldap_host: example.com
+    ldap_port: 123
+    ldap_use_ssl: false
+    ldap_insecure_skip_verify: true
+    ldap_user_filter: (cn=%s)
+    ldap_group_filter: (cn=%s)
+    ldap_attributes: ['memberOf', 'cn', 'mail']
+    ldap_base: DC=example,DC=com
+    ldap_bind_dn: CN={USERID},OU=xyz,OU=Users,OU=Accounts,DC=example,DC=com
+    ldap_search_filter: (&(name={USERID}))
+    # Time for token expiration
```

Comment:
See my comments in the related Kiali PR. This token expiration is not used, remove it. Also the "# AD  IP host and port" comment - the AD is confusing (what if I'm not using Active Directory? Is that what AD means? Does this LDAP feature only support AD? etc etc.) ... It also is obvious what host and port are. So that comment can be removed.

Reply:
You might also want to add some comments before the example YAML explaining some of the more interesting and important ones or ones that are not obvious (like ldap_role_filter, ldap_search_filter, ldap_bind_dn, etc).